source/jormungandr/jormungandr/scenarios/ridesharing/ouestgo.py

```diff
@@ -200,7 +177,7 @@ def _request_journeys(self, from_coord, to_coord, period_extremity, instance_par
         """
         dep_lat, dep_lon = from_coord.split(',')
         arr_lat, arr_lon = to_coord.split(',')
-
+        timezone = get_default_timezone()
```

Comment:
It deserves a comment that TZ for mindate/maxdate is the coverage's one, or Paris IMO:
```suggestion
        timezone = get_default_timezone()  # using coverage's TZ (or Paris) for mindate and maxdate
```

Reply:
:+1: